components/write_to_file/fondant_component.yaml

```diff
@@ -0,0 +1,23 @@
+name: Write to file
+description: >-
+  A Fondant component to write a dataset to file on a local machine or to a cloud storage bucket.
+  The dataset can be written as csv or parquet.
+image: 'fndnt/write_to_file:dev'
+tags:
+  - Data writing
+
+consumes:
+  additionalProperties: true
+
+args:
+  path:
+    description: >- 
+      Path to store the dataset, whether it's a local path or a cloud storage bucket, 
+      must be specified. A separate filename will be generated for each partition.
+      If you are using the local runner and export the data to a local directory, 
+      ensure that you mount the path to the directory using the `--extra-volumes` argument.
+    type: str
+  format:
+    description: Format for storing the dataframe can be either `csv` or `parquet`.
+    type: str
+    default: csv
```

Comment:
I might have overlooked it but I would clearly document that `csv` is the default.

Also IMO parquet > csv in almost every way (except you need a tool to read parquet)